config/config.go

```diff
@@ -75,26 +77,21 @@ func getHomeDir() string {
 func Load(configFilePath string) (Config, error) {
```

Comment:
Do you want to make this private so that people will be forced to use configManager to load the config that comes with watcher configured? Otherwise if people use this method, they will loose the ability to read the config file path from default environment var. Or you can invoke ioutil.ReadFile(getConfigurationFile(configFilePath)) so that all cases are covered

Reply:
Created #267 to follow up on this.